pages/common/gh-pr.md

```diff
@@ -0,0 +1,24 @@
+# gh-pr
+
+> Work seamlessly with GitHub from the command line.
+> More information: <https://cli.github.com/>.
+
+- Create a pull request:
+
+`gh pr create`
+
+- Check out pull requests locally:
+
+`gh pr checkout {{pr_number}}`
+
+- View the changes made in the PR:
+
+`gh pr diff`
+
+- Approve the pull request of the current branch:
+
+`gh pr review --approve`
+
+- Merge a PR if you're on the branch, removes the branch on local and remote:
```

Comment:
Perhaps this will read better?
```suggestion
- Merge the pull request associated with the current branch, removing the branch on both the local and the remote:
```

Reply:
Updated the PR